src/conduit-lwt-unix/conduit_lwt_unix_ssl.dummy.mli

```diff
@@ -31,6 +34,8 @@ module Client : sig
     ?ctx:[ `Ssl_not_available ] ->
     ?src:Lwt_unix.sockaddr ->
     ?hostname:string ->
+    ?ip:Ipaddr.t ->
+    ?verify:verify ->
```

Comment:
I wonder, shouldn't `?verify` be implicit? If`~ip` is provided, likely `ip` in the verify record will be true, if `~hostname` is provided, likely `hostname` in the verify record will be true.

Is there a use case where both `hostname` and `ip` are provided and should be verified? Is there a use case where `hostname` is provided but shouldn't be verified?

Reply:
> I wonder, shouldn't ?verify be implicit? 

Unfortunately, no. hostname can easily be a string representing an IP, which makes the peers' certificates unverifiable.

Regarding IP, it's rare that certificates contain IP addresses while this call will also contain an IP when using cohttp if I understood it correctly, so it's going to be rare the case where it needs to be verified.

Since both verifications are needed case-by-casse bases I think it's better if it's explicit with a reasonable default. Having them implicit wouldn't work with xenserver's use-case